contracts/eosio.bios/src/eosio.bios.cpp

```diff
@@ -54,4 +57,82 @@ void bios::reqactivated( const eosio::checksum256& feature_digest ) {
    check( is_feature_activated( feature_digest ), "protocol feature is not activated" );
 }
 
+void bios::syscurrency() {
+    require_auth(get_self());
+
+    // Set the system resource currency to oncoin.tmy
+    eosio::symbol system_resource_currency = eosio::symbol("ONO", 4);  
+
+    // Set the resource token to oncoin.tmy
+    name resource_token_contract = eosio::name("oncoin.tmy");
+}
+
+
+void bios::setramprice(double new_price) {
```

Comment:
I think it makes more sense to use a `setresourceparams(double ram_price, uint84_t total_ram_available) all` which also updates `total_ram_available` at the same time

Reply:
bigen error (Name {setresourceparams} is more than 13 characters long)
       [[eosio::action]] void setresourceparams(double ram_price, uint64_t total_ram_available); 


so i use setresparams